app/code/Magento/Inventory/Test/Integration/Source/DefaultSource.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Test\Integration\Source;
+
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+use Magento\Inventory\Model\Source;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class DefaultSource
+ * @package Magento\Inventory\Test\Integration\Source
+ */
+class DefaultSource extends TestCase
```

Comment:
@maghamed 
What do think, maybe is better to create WebApi test instead of Integration?
Because Default Stocks looks like part of api provided CatalogInventory bridge module

Reply:
@naydav  yes, web api test in InventoryCatlog module makes sense